.storybook/main.js

```diff
@@ -17,9 +19,49 @@ const replaceBabelWithSwc = (config) => {
     return config;
 };
 
+const addVanillaExtractSupport = (config) => {
+    config.module.rules.forEach((rule) => {
+        if (
+            rule.test &&
+            (Array.isArray(rule.test)
+                ? rule.test.some((exp) => exp.test('random.css'))
+                : rule.test.test('random.css'))
+        ) {
+            const previousExclude = rule.exclude || [];
+            rule.exclude = [
+                ...(Array.isArray(previousExclude) ? previousExclude : [previousExclude]),
+                /\.vanilla\.css$/,
+            ];
+        }
+    });
```

Comment:
can you add a comment explaining why is this needed?

Reply:
added